test/src/test/java/org/corfudb/integration/LogReplicationAbstractIT.java

```diff
@@ -339,6 +457,99 @@ public void checkpointAndTrim(boolean active, List<CorfuTable> tables) {
         checkpointAndTrimCorfuStore(cpRuntime, trimMark);
     }
 
+    public void verifyDataOnStandby(int expectedConsecutiveWrites) {
+        for(Map.Entry<String, Table<Sample.StringKey, Sample.IntValue, Sample.Metadata>> entry : mapNameToMapStandby.entrySet()) {
+
+            log.debug("Verify Data on Standby's Table {}", entry.getKey());
+
+            // Wait until data is fully replicated
+            while (entry.getValue().count() != expectedConsecutiveWrites) {
+                // Block until expected number of entries is reached
+            }
+
+            log.debug("Number updates on Standby Map {} :: {} ", entry.getKey(), expectedConsecutiveWrites);
+
+            // Verify data is present in Standby Site
+            assertThat(entry.getValue().count()).isEqualTo(expectedConsecutiveWrites);
+
+            for (int i = 0; i < (expectedConsecutiveWrites); i++) {
+                assertThat(entry.getValue().get(Sample.StringKey.newBuilder().setKey(String.valueOf(i)).build()).getPayload()).isNotNull();
+            }
+        }
+    }
+
+    /**
+     * Checkpoint and Trim Data Logs
+     *
+     * @param active true, checkpoint/trim on active cluster
+     *               false, checkpoint/trim on standby cluster
+     */
+    public void checkpointAndTrim(boolean active) {
```

Comment:
looks like this is being used for UFO, can we rename it accordingly so that it doesnt confuse with the existing cpAndTrim() variant?

Reply:
private?